src/snowflake/cli/_plugins/nativeapp/version/version_processor.py

```diff
@@ -328,6 +327,9 @@ def process(
                     "Manifest.yml file does not contain a value for the version field."
                 )
 
+        # Make the version a valid identifier, adding quotes if necessary
+        version = to_identifier(version)
```

Comment:
any way we can unify this to get_version() in manager or processor so we don't have to remember to do it every time, similar to get_app() or get_package() ?

Reply:
this also needs to handle the string passed in by the user, it's not part of the project model. I could change this to a CLI callback that casts it at that point though